packages/engines/README.md

```diff
@@ -0,0 +1,10 @@
+# `@prisma/engines`
+
+This package ships the Prisma Engines, namely the Query Engine, Migration Engine, Introspection Engine and Prisma Format.
+It has a one to one mapping to the engine releases in its version.
```

Comment:
This is not true anymore, the version will be monorepo style

Reply:
We could add a note instead that it gets the actual version of the engines via `@prisma/engines-version` which follows a different versioning pattern that includes that information as documented here currently.